runs/2021/pyladies-ostrava-podzim/info.yml

```diff
@@ -69,30 +69,30 @@ plan:
 - slug: expressions
   title: 'Výrazy a podmínky'
   date: 2021-09-22
-#   materials:
-#   - url: https://youtu.be/r0esiaghzrY?t=483
-#     title: (1:49) Neuzavřená závorka – video (+ domácí úkoly v první části videa)
-#   - lesson: adm/close-paren
-#
-#   - url: https://youtu.be/9aJcvOfleFs
-#     title: (7:50) První program – video
-#   - lesson: beginners/hello-world
-#
-#   - url: https://youtu.be/14-LVG9Edng
-#     title: (12:38) Print a chybové hlášky – video
-#   - lesson: beginners/print
-#
-#   - url: https://youtu.be/-zKws24FmCg
-#     title: (15:08) Proměnné – video
-#   - lesson: beginners/variables
-#
-#   - url: https://youtu.be/nNXDXuN6Smg
-#     title: (8:32) Vyhodnocování výrazů – video
-#   - lesson: beginners/expressions
-#
-#   - url: https://youtu.be/Q1YJqWzfnck
-#     title: (17:27) Porovnání – video
-#   - lesson: beginners/comparisons
+  materials:
+  - url: https://youtu.be/r0esiaghzrY
+    title: (9:52) Neuzavřená závorka – video (+ domácí úkoly v první části videa)
+  - lesson: adm/close-paren
+
+  - url: https://youtu.be/9aJcvOfleFs
+    title: (7:50) První program – video
+  - lesson: beginners/hello-world
+
+  - url: https://youtu.be/14-LVG9Edng
+    title: (12:38) Print a chybové hlášky – video
+  - lesson: beginners/print
+
+  - url: https://youtu.be/-zKws24FmCg
+    title: (15:08) Proměnné – video
+  - lesson: beginners/variables
+
+  - url: https://youtu.be/nNXDXuN6Smg
+    title: (8:32) Vyhodnocování výrazů – video
+  - lesson: beginners/expressions
```

Comment:
One thing about this part of the lesson, in the video, the final expression is a bit different than the one in the text version. So maybe let the attendees know, that the expression is a bit different in the text and in the video version.

Reply:
Do you think they will notice? I mean, we already mentioned that the video and text might not be 100% the same and I honestly think that attendees use either video or text but not both. But yes, it makes sense to repeat that at the first meeting.

Reply:
I am not sure, how many attendees will notice it. The only things, so far, that I found that don't match 100% are the lesson numbers in the materials and our plan and the expression in this lesson.
But mentioning it at the first meeting seems like a solution for it.

Reply:
There'll be more cases like this in the future. It's hard for me to stick to a script :)